### PR TITLE
Add ros_arguments option to Node action

### LINKED ISSUE #60: Add `ros_arguments` to `Node` action constructor
#### Feature description
See https://github.com/ros2/rmw_fastrtps/issues/265#issuecomment-526181148.

### PR DESCRIPTION
**Pull request by christophebedard**
Closes #60

This adds a `ros_arguments` option to the `Node` action. It's simply: `ros_arguments == ['--ros-args'] + arguments`.

Signed-off-by: Christophe Bedard <bedard.christophe@gmail.com>

### CODE REVIEW

File: launch_ros/launch_ros/actions/node.py
Comment (hidmic) [inline launch_ros/launch_ros/actions/node.py:185]: @christophebedard I see the convenience BUT is it necessary? Most ROS arguments can be given in an idiomatic way  (e.g. via `remappings`) already. If we need `--log-level`, I'd argue we should add it. CC @wjwwood @ivanpauno.

Comment (christophebedard) [reply launch_ros/launch_ros/actions/node.py:185]: I found #60 while answering an answers.ros.org question: https://answers.ros.org/question/380883/specify-log-level-on-ros2-node-in-launch-file/#380965. In this case, it seemed to confuse the person. They weren't sure whether they needed to use `--` beause `launch_ros` appends `--ros-args ...` too (even if in practice `rcl` handles it just fine). I think having `ros_arguments` would make it much simpler, but yeah it's up to you the maintainers :stuck_out_tongue_closed_eyes:

> If we need `--log-level`

you mean like a `LogLevel` substitution?

Comment (hidmic) [reply launch_ros/launch_ros/actions/node.py:185]: > I found #60 while answering an answers.ros.org question: https://answers.ros.org/question/380883/specify-log-level-on-ros2-node-in-launch-file/#380965. In this case, it seemed to confuse the person.

I see. I was wondering about the use case. If not having to write out `--ros-args` helps people, I'm onboard.

> you mean like a LogLevel substitution?

Like a `log_level` argument that can be a substitution. But that's a separate PR.

Comment (wjwwood) [reply launch_ros/launch_ros/actions/node.py:185]: Seems like we can handle things better if the user explicitly tells us when something is a ros argument, so +1 from me.

**Comment by christophebedard** [comment]
CI `--packages-above launch_ros test_launch_ros`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14817/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9567/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12483/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/14972/)

**Review by hidmic — APPROVED** [review]
LGTM

**Comment by christophebedard** [comment]
@hidmic I can't merge PRs so I'll leave that to you!

**Comment by hidmic** [comment]
Going in ! Thanks for the patience @christophebedard.

**Comment by christophebedard** [comment]
> Argh, @christophebedard one thing we missed is adding support for this attribute in the `Node.parse` function. Would you mind?

oh yeah I really didn't think about that! I'll create another PR as a follow-up.

**Pull request merged**
